makeflow/src/dag_file.c

```diff
@@ -84,4 +87,32 @@ int dag_file_in_trans( const struct dag_file *f )
 		return 0;
 }
 
+int dag_file_mount_clean(struct dag *d) {
```

Comment:
I like the idea that all the malloced space should be freed somewhere.
If this does not break any other part, I would suggest leaving them here.